File: StarWars/APICORE/Entities/Credit_Card.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace APICORE.Entities
{
    [Table("CreditCard_Transaction")]
    public class Credit_Card
    {
        [Key]
        public int Id { get; set; }
        public int TransactionId { get; set; }
        public string Card_number { get; set; }
        public int Value { get; set; }
        public int Cvv { get; set; }
        public string Card_holder_name { get; set; }
        public string Exp_date { get; set; }
        public virtual Transaction Transaction { get; set; }
    }
}

File: desafiostone/src/main/java/br/com/stone/desafiostone/entity/CreditCard.java
package br.com.stone.desafiostone.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "tb03_creditcard")
@Getter
@Setter
public class CreditCard {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private String card_number;
    private double value;
    private int cvv;
    private String card_holder_name;
    private String exp_date;
  
}
